### PR TITLE
feat: Homepage responsive design

### LINKED ISSUE #5857: User experience - New look and feel for Composer-Home
**Composer Welcome / Start Page** 

Refresh of Composer Home to align with our Build release. 

References:

- Visual Studio - http://dotnetmirror.com/Resources/Uploaded-Files/fe0c4c6e-4fcc-49b6-bb35-376b009317b2/visual-studio-2017-Start-Page.png  
- Visual Studio Code - 2019-02-28-14_44_29-Welcome-Visual-Studio-Code.png (1912×1040) (qlikfix.com) 
- [See also (Gary's email/doc): ](https://nam06.safelinks.protection.outlook.com/ap/w-59584e83/?url=https%3A%2F%2Fmicrosoft.sharepoint.com%2F%3Aw%3A%2Ft%2FConversationalAI785%2FEQ95DgvK6eRBn0fK_mMZMAMBNuzGFM9oGUN6K0JUqUoYnQ%3Fe%3DIDwQi4&data=04%7C01%7Cmariekei%40microsoft.com%7C09e7a3a263cf4969308c08d8d68e782d%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637495254882607416%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=C2vmNkGB1rfGgfii%2FLEPMeoVBw08z95Dz8%2By5dtaKPA%3D&reserved=0)

Content 
- Recent bots list 
- Featured videos 
-  Tutorial

Functionality/tasks
- New bot 
- Open existing bot 
- Open/install Emulator

Access to:
- What’s new 
- Release notes
- Composer docs (Learn / Help) - Short list (maybe 3?) of links (title & brief sub title), blog posts, preview features etc.  
- Composer GitHub repo 
-  Bot Builder Community?  (Given our likely push for package building?) 
-  Stack Overflow 

Related products?
- PVA?
- Health bot?
- LUIS?
- QnA?
- AI Studio?

**Remove from current design:**
- examples list – 

**Future idea**
- Bundle Emulator
- Pin your favorite projects to the home screen

### PR DESCRIPTION
**Pull request by zhixzhan**
## Description

Follow up PR #6642 , polish UX design

## Task Item
close #5857

### CODE REVIEW

File: Composer/packages/server/src/models/bot/feedManager.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import axios from 'axios';
 
 // TODO: update feed url
-const feedUrl = 'https://aka.ms/bf-composer-home-feed-v1';
+const feedUrl = 'https://api.jsonbin.io/b/6066712c7474c23d9027bf10/10';
```
Comment (zhixzhan) [inline Composer/packages/server/src/models/bot/feedManager.ts:6]: modified for test

**Comment by zhixzhan** [comment]
The dimension of image cover would be recommend at  ```245 x 95```  px or  higher resolution  at  ```490 x 190``` px

too short will apply a blur effect background to fill the blank space

**Comment by zhixzhan** [comment]
feed content update in https://github.com/microsoft/BotFramework-Composer/pull/6829

**Review by beyackle — DISMISSED** [review]

File: Composer/packages/client/src/components/Header.tsx [hunk 2]
```diff
@@ -52,6 +52,7 @@ const headerContainer = css`
   background: ${SharedColors.cyanBlue10};
   height: 50px;
   display: flex;
+  min-width: 1200px;
```
[the remark this answers is not in the document]
Comment (zhixzhan) [reply Composer/packages/client/src/components/Header.tsx]: It's better to set on header left part.

File: Composer/packages/client/src/pages/home/CardWidget.tsx [hunk 3]
```diff
@@ -2,22 +2,114 @@
...
+    if (state === ImageLoadState.error) {
+      setUsedImageCover(defaultImageCover);
+    }
+  };
+
+  // detect image cover dimention to decide apply blur background or not.
+  const onImgLoaded = (e) => {
+    const rect = e.currentTarget.getBoundingClientRect();
+    const whRatio = rect.width / rect.height;
+    if (whRatio < 1.5) {
```
[the remark this answers is not in the document]
Comment (zhixzhan) [reply Composer/packages/client/src/pages/home/CardWidget.tsx]: updated

File: Composer/packages/client/src/pages/home/Home.tsx [hunk 4]
```diff
@@ -141,9 +142,9 @@ const Home: React.FC<RouteComponentProps> = () => {
   ];
   return (
     <div css={home.outline}>
-      <h1 css={home.title}>{formatMessage(`Bot Framework Composer`)}</h1>
       <div css={home.page}>
         <div css={home.leftPage} role="main">
+          <h1 css={home.title}>{formatMessage(`Bot Framework Composer`)}</h1>
```
[the remark this answers is not in the document]
Comment (zhixzhan) [reply Composer/packages/client/src/pages/home/Home.tsx]: It's a mistake, fixed now, thanks

**Review by boydc2014 — APPROVED** [review]

**Pull request merged**